packages/react-router/index.js

```diff
@@ -476,7 +476,10 @@ export function useRoutes(routes, basename = '', caseSensitive = false) {
 
   basename = basename ? joinPaths([parentPathname, basename]) : parentPathname;
 
-  let location = useLocation();
+
```

Comment:
```suggestion
```
Extraneous space.

Reply:
Not sure how this space came here. I will remove it.